### PR TITLE
feat: Pachyderm Storage

### PR DESCRIPTION
**Pull request by makseq**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -15,6 +15,18 @@ RUN set -eux; \
     build-essential postgresql-client libmysqlclient-dev mysql-client python3.8 python3-pip python3.8-dev \
     uwsgi git libxml2-dev libxslt-dev zlib1g-dev
 
+# Install fuse and pachctl
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends curl fuse3\
```
Comment (farioas) [inline Dockerfile:20]: Merge it with upper layer

File: Dockerfile [hunk 2]
```diff
@@ -15,6 +15,18 @@ RUN set -eux; \
...
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN curl -o /tmp/pachctl.deb -L https://github.com/pachyderm/pachyderm/releases/download/v2.0.8/pachctl_2.0.8_amd64.deb\
```
Comment (farioas) [inline Dockerfile:24]: Make version configurable, eg expose it as `ENV PATCHCTL_VERSION=2.0.8`.

Delete deb package from tmp after install

File: Dockerfile [hunk 3]
```diff
@@ -15,6 +15,18 @@ RUN set -eux; \
...
+ && dpkg -i /tmp/pachctl.deb
+
+RUN mkdir /pfs
+COPY deploy/init-pachyderm.sh .
```
Comment (farioas) [inline Dockerfile:28]: Not required, because of `COPY . /label-studio`

**Review by farioas — CHANGES_REQUESTED** [review]

**Comment by farioas** [comment]
/git merge develop
> **Error**: failed to execute "merge" command
>
> [Workflow run](https://github.com/heartexlabs/label-studio/actions/runs/4418021963)

**Review by farioas — CHANGES_REQUESTED** [review]
Requested changes from the previous review were not resolved.

**Comment by robot-ci-heartex** [comment]
This PR is stale because it has been open 45 days with no activity. Remove `stale` label or comment or this will be closed in 10 days.

**Comment by robot-ci-heartex** [comment]
This PR was closed because it has been stalled for 10 days with no activity.

**Pull request closed without merging**